src/Symfony/Bundle/FrameworkBundle/Resources/config/debug.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\DependencyInjection\Loader\Configurator;
+
+use Symfony\Component\HttpKernel\Controller\ArgumentResolver\NotTaggedControllerValueResolver;
+use Symfony\Component\HttpKernel\Controller\TraceableArgumentResolver;
+use Symfony\Component\HttpKernel\Controller\TraceableControllerResolver;
+use Symfony\Component\HttpKernel\Debug\TraceableEventDispatcher;
+
+return static function (ContainerConfigurator $container) {
+    $container->services()
+        ->set('debug.event_dispatcher', TraceableEventDispatcher::class)
+            ->decorate('event_dispatcher')
+            ->tag('monolog.logger', ['channel' => 'event'])
+            ->tag('kernel.reset', ['method' => 'reset'])
```

Comment:
let's place tag after args and calls, ref https://github.com/symfony/symfony/pull/37211#issuecomment-642639486